test/Faker/Provider/InternetTest.php

```diff
@@ -117,4 +117,50 @@ public function testMacAddress()
     {
         $this->assertRegExp('/^([0-9A-F]{2}[:]){5}([0-9A-F]{2})$/i', Internet::macAddress());
     }
+
+    public function test_htmlLoremDefault()
+    {
+        $html = Internet::html();
+        $this->assertTrue($this->containsHtml($html));
+    }
+
+    public function test_htmlLoremSmallLengths()
+    {
+        $plus5 = Internet::html(Internet::SMALLEST_HTML_LENGTH+5);
+        $this->assertTrue($this->containsHtml($plus5));
+    }
+
+    public function test_htmlTooSmall()
+    {
+        $this->setExpectedException('InvalidArgumentException');
+        Internet::html(Internet::SMALLEST_HTML_LENGTH + 4);
+        Internet::html(Internet::SMALLEST_HTML_LENGTH - 1);
```

Comment:
Which one is supposed to throw an exception, the first or the second? Perhaps you need two tests?